src/app/components/main-menu/main-menu.directive.js

```diff
@@ -89,9 +95,22 @@ class MainMenuController {
     this.socketService.emit('getMenuItems');
   }
 
+  authInit() {
+    this.$scope.$watch(() => this.authService.user, (user) => {
+      this.user = user;
+    });
+  }
+
   initMenuListSource() {
     this.$scope.$watch( () => this.browseService.sources , (sourcesData) => {
       this.sources = sourcesData;
+      if (sourcesData) {
```

Comment:
This is an inaccurate way to determine if we have a premium account. We can set up BE calls if needed